packages/tools/fetch-tool/src/fluidAnalyzeMessages.ts

```diff
@@ -425,7 +427,8 @@ class SummaryAnalyzer implements IMessageAnalyzer {
 			this.lastSummaryOp = message.sequenceNumber;
 		}
 		if (message.type === MessageType.SummaryAck || message.type === MessageType.SummaryNack) {
-			const contents: ISummaryProposal = message.contents.summaryProposal;
+			const summaryMessage = message as ISummaryAckMessage | ISummaryNackMessage;
+			const contents: ISummaryProposal = summaryMessage.contents?.summaryProposal;
```

Comment:
Shouldn't `contents` always be defined on `ISummaryAckMessage | ISummaryNackMessage`?
```suggestion
			const contents: ISummaryProposal = summaryMessage.contents.summaryProposal;
```